kasa/feature.py

```diff
@@ -140,22 +139,24 @@ def value(self):
             raise ValueError("Not an action and no attribute_getter set")
 
         container = self.container if self.container is not None else self.device
-        if isinstance(self.attribute_getter, Callable):
+        if callable(self.attribute_getter):
             return self.attribute_getter(container)
         return getattr(container, self.attribute_getter)
 
-    async def set_value(self, value):
+    async def set_value(self, value: int | float | bool | str | Enum | None) -> Any:
```

Comment:
This could be a type alias, and we probably need to set the same for the getter.

Reply:
It does fix the type errors in the HA integration however adding type hints to the getter actually causes mypy errors in HA because the type is broader that the things it's being set to (mypy doesn't currently check that properties are typed).  This is the minimal change at the moment without introducing some sort of generic typing solution.

N.B. I decided not to use a type alias it was only used in one place.

Reply:
Ok, that's fine :+1: